src/main/java/chess/dao/ChessLogDao.java

```diff
@@ -0,0 +1,109 @@
+package chess.dao;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ChessLogDao {
+    private static final String MOVE = "move ";
+    private static final String DELIMITER = " ";
+
+    public Connection getConnection() {
+        Connection con = null;
+
+        loading();
+        return connection(con);
+    }
+
+    private void loading() {
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            System.err.println(" !! JDBC Driver load 오류: " + e.getMessage());
+            e.printStackTrace();
+        }
+    }
+
+    private Connection connection(Connection connection) {
+        String server = "localhost:13306"; // MySQL 서버 주소
+        String database = "chess"; // MySQL DATABASE 이름
+        String option = "?useSSL=false&serverTimezone=UTC";
+        String userName = "root"; //  MySQL 서버 아이디
+        String password = "root"; // MySQL 서버 비밀번호
+
+        try {
+            connection = DriverManager.getConnection("jdbc:mysql://" + server + "/" + database + option, userName, password);
+            System.out.println("정상적으로 연결되었습니다.");
+        } catch (SQLException e) {
+            System.err.println("연결 오류:" + e.getMessage());
+            e.printStackTrace();
+        }
+
+        return connection;
+    }
+
+    public void closeConnection(Connection con) {
+        try {
+            if (con != null)
+                con.close();
+        } catch (SQLException e) {
+            System.err.println("con 오류:" + e.getMessage());
+        }
+    }
+
+    public void addLog(String roomId, String target, String destination) {
+        Connection connection = getConnection();
+        String query = "INSERT INTO chessgame (room_id, target, destination) VALUES (?, ?, ?)";
+        try (PreparedStatement pstmt = connection.prepareStatement(query);) {
```

Comment:
```suggestion
        try (Connection connection = getConnection(); PreparedStatement pstmt = connection.prepareStatement(query)) {
```
Connection, PreparedStatement는 AutoClose 구현체이기 때문에 이렇게 사용 가능합니다 :) 
쿼리문 위에 `Connection connection = getConnection();` 는 지워주셔도 됩니다